crates/primitives/src/op_mainnet.rs

```diff
@@ -0,0 +1,54 @@
+//! Helpers for working with Optimism.
+
+/* --------------------- REPLAYED TRANSACTIONS IN BLOCKS BELOW BEDROCK --------------------- */
```

Comment:
it would be nice to include this context in the `//!` module documentation rather than as a single line comment, so it can be visible in local docs pages. It could also link to the replayed tx list or individual tx consts